akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/EventStreamTest.java

```diff
@@ -12,7 +12,6 @@
   static class SomeClass {}
 
   public static void compileOnlyTest(ActorSystem<?> actorSystem, ActorRef<SomeClass> actorRef) {
-    actorSystem.eventStream().tell(Subscribe.of(SomeClass.class, actorRef));
-    actorSystem.eventStream().tell(new Subscribe<>(SomeClass.class, actorRef));
+    actorSystem.eventStream().tell(new EventStream.Subscribe<>(SomeClass.class, actorRef));
```

Comment:
No coverage of Subscribe anymore?

Reply:
`Subscribe.of` was removed, `new EventStream.Subscribe, is still covered here